Source/Core/VideoCommon/PerformanceMetrics.h

```diff
@@ -21,20 +30,31 @@ class PerformanceMetrics
   void CountFrame();
   void CountVBlank();
 
+  void CountThrottleSleep(DT sleep);
+  void CountPerformanceMarker(Core::System& system, s64 cyclesLate);
+
   // Getter Functions
   double GetFPS() const;
   double GetVPS() const;
   double GetSpeed() const;
+  double GetMaxSpeed() const;
 
   double GetLastSpeedDenominator() const;
 
   // ImGui Functions
-  void DrawImGuiStats(const float backbuffer_scale) const;
+  void DrawImGuiStats(const float backbuffer_scale);
 
 private:
-  PerformanceTracker m_fps_counter{"render_times.txt"};
-  PerformanceTracker m_vps_counter{"vblank_times.txt"};
+  PerformanceTracker m_fps_counter{"render_times.txt", std::nullopt};
+  PerformanceTracker m_vps_counter{"vblank_times.txt", std::nullopt};
```

Comment:
This isn't really a problem, but... why did you add the `std::nullopt` here? It's already the default parameter.